config/samples/layout/edge/glance_v1beta1_glance.yaml

```diff
@@ -4,25 +4,28 @@ metadata:
   name: glance
 spec:
   serviceUser: glance
-  customServiceConfig: |
-    [DEFAULT]
-    debug = true
   databaseInstance: openstack
   databaseAccount: glance
-  keystoneEndpoint: central
+  keystoneEndpoint: default
   glanceAPIs:
-    edge1:
-      preserveJobs: false
+    default:
+      customServiceConfig: |
+        [DEFAULT]
+        enabled_backends=default_backend:rbd,az1:rbd,az2:rbd
```

Comment:
I have to review the samples config because this is not going to work w/o a ceph Pod deployed by in the kuttl context